jvb/src/main/kotlin/org/jitsi/videobridge/service/ServiceSupplier.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.videobridge.service
+
+interface ServiceSupplier<T> {
```

Comment:
Why is this needed in addition to `java.util.function.Supplier`?

Reply:
This was a leftover from a prior implementation and was removed in later commits